docs/source/conf.py

```diff
@@ -584,6 +587,9 @@
 #
 # texinfo_no_detailmenu = False
 
+# -- Options for mermaid output -------------------------------------------
+
+mermaid_output_format = 'svg'
```

Comment:
Is there a reason we are setting this here, instead of letting it generate the svg image live through the js mermaid script? (the default setting)

That would avoid requiring mermaid-cli for building the docs

Reply:
@zeroshade Why do you choose `svg` here? I just used your code here.